mysql-test/suite/innodb/t/percona_crash_recovery_innodb_status.test

```diff
@@ -0,0 +1,29 @@
+--source include/have_debug.inc
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+
+--echo #
+--echo # Test printing of InnoDB status during crash recovery
+--echo #
+
+--let SEARCH_FILE= $MYSQLTEST_VARDIR/log/my_restart.err
+
+CREATE TABLE t1 (a INT PRIMARY KEY) ENGINE=InnoDB;
+
+BEGIN;
+INSERT INTO t1 VALUES (1);
+
+--source include/kill_mysqld.inc
+--error 137
```

Comment:
From `log_file_size.test`
```# On Windows, DBUG_SUICIDE() does exit(3) instead of SIGKILL (128+9).```
So (just in case one day we support Windows)
`--error 137,3`